### DISCUSSION TITLE
Dynamic Dashboard based on user input

### DISCUSSION THREAD

**Discussion by alex1404** [apache/superset#23095] [General]
Hi all,
I'm superset beginner and I would like to know if Superset supports such use case

Use case:
A dashboard which contains a pie chart and user input field as threshold where user can insert a numeric value as a threshold and as a result the pie chart will be updated accordingly ( quite similar to what-if scenario in excel)

 I wrote this query to emphasize the dataset query 

```
SELECT 'Above threshold' AS title,  SUM(CASE WHEN "DISTANCE" <**{{user input }}** THEN 1 ELSE 0 END)/cast( count(*)  AS DECIMAL(30, 20)) as counter  FROM FLIGHTS
union 
SELECT 'Below threshold' AS title,  1- (SUM(CASE WHEN "DISTANCE" <**{{ user input }}** THEN 1 ELSE 0 END)/cast( count(*)  AS DECIMAL(30, 20)))  FROM FLIGHTS  
```
for example  in case user set the threshold to 2000 the result is
![image](https://user-images.githubusercontent.com/12210275/219384497-3b005dc8-2def-4ff8-93db-71f32de70784.png)

and for threshold of 1000 the result is 
![image](https://user-images.githubusercontent.com/12210275/219385101-a84206ac-f633-4c5e-8e94-f2ca644a6a1e.png)

Any idea how to create such dashboard??? 
Thanks
Alex

**Comment by alex1404** [comment]
Anyone???

**Comment by dk132001** [comment]
Hi i too worked on this kind of requirement did u got solution?
